site/de/phase3.md

```diff
@@ -17,15 +18,68 @@ Das Team gab eine Einführung in die [Ziele und öffentlichen Kommunikationskan
+
+<br/><br/>
+Weitere Informationen: [Projektseite der Deutschen Forschungsgemeinschaft](https://gepris.dfg.de/gepris/projekt/460554747?language=de)
 
 
 ## Modulprojekte
-* [Workflow für werkspezifisches Training auf Basis generischer Modelle mit OCR-D sowie Ground Truth Aufwertung (UB Mannheim)](https://www.bib.uni-mannheim.de/ihre-ub/projekte-der-ub/ocr-d-modelltraining/)
-* [Font Group Recognition for Improved OCR (JGU Mainz, FAU Erlangen-Nürnberg)](https://lme.tf.fau.de/news/dfg-funds-font-group-recognition-for-improved-ocr/)
-* [OLA-HD Service - Ein generischer Dienst für die Langzeitarchivierung historischer Drucke (SUB Göttingen, GWDG)](https://gepris.dfg.de/gepris/projekt/460652320?language=de)
+
+### Workflow für werkspezifisches Training auf Basis generischer Modelle mit OCR-D sowie Ground Truth Aufwertung 
+_UB Mannheim_
+
+Ziel dieses Projektes ist, dass Einrichtungen (zum Beispiel Bibliotheken) möglichst einfach die Module des OCR-D-Workflows nachtrainieren können, so dass bessere Erkennungs­raten für spezifische Werke erreicht werden können.
+
+<br/><br/>
+Weitere Informationen: [Projektseite der Universitätsbibliothek Mannheim](https://www.bib.uni-mannheim.de/ihre-ub/projekte-der-ub/ocr-d-modelltraining/)
+
+### Erkennung von Schriftartgruppen zur OCR Verbesserung
+_JGU Mainz, FAU Erlangen-Nürnberg_
+
+<br/><br/>
+Weitere Informationen: [Projektseite der Deutschen Forschungsgemeinschaft](https://gepris.dfg.de/gepris/projekt/460605811?language=de)
+
+### OLA-HD Service – Ein generischer Dienst für die Langzeitarchivierung historischer Drucke 
+_SUB Göttingen, GWDG_
+
+Das primäre Projektziel ist die Entwicklung eines produktiven Dienstes für die Langzeitarchivierung von historischen Drucken im Rahmen von OCR-D. Dieser OLA-HD Service baut auf dem entsprechenden Prototypen aus der OCR-D Phase II auf, erweitert diesen gemäß der Anforderungen der Implementierungsprojekte, wird in das OCR-D Framework integriert und wird entsprechend der Ausschreibungsanforderungen generisch konzipiert und umgesetzt.
+
+<br/><br/>
```

Comment:
Great, maybe let the projects know about these more extensive descriptions in the lobby, to make sure they're accurate?

Reply:
The descriptions were provided by the projects for this purpose so I hope it's okay but it's a good idea to let them know that it's online now and that they can change them by doing a PR